app/js/components/setsOrderEntry/OrderSetForm.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import SelectBox from '../selectBox';
+import Accordion from '../orderEntry/Accordion';
+import OrderHeader from '../orderEntry/OrderHeader';
+import OrderSetDetails from './OrderSetDetails';
+import './styles.scss';
+
+
+class OrderSetForm extends PureComponent {
+  state = {
+    items: [{ value: 'Choose a set to add', id: 0 }, { value: 'Mineral', id: 1 }],
+    selectedItem: '',
+    isDisabled: false,
+    displaySets: 'block',
+    displayForm: 'none',
+  }
+
+  onSelectOrderSet = () => {
+    this.setState({ displaySets: 'none', displayForm: 'block' });
+  }
+
+  selectBoxOnchange = () => {
+    this.onSelectOrderSet();
+  }
```

Comment:
I think this method is unnecessary cause it does nothing but calls another method that can be called directly.
so discard this method and call the `onSelectOrderSet` method directly.

Reply:
Done